src/styles/index.css

```diff
@@ -0,0 +1,107 @@
+body {
+  margin: 0;
+  font-family: Roboto, sans-serif;
+}
+
+.card {
+  width: 200px;
+  padding: 16px;
+}
+
+.card__image {
+  width: 160px;
+  display: flex;
+  justify-content: center;
+  margin-left: 20px;
+}
+
+.card__name {
+  display: flex;
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 18px;
+  color: #060b35;
+  justify-content: center;
+  margin-top: 40px;
+  margin-bottom: 0;
+}
+
+.card__product--code {
+  font-size: 10px;
+  line-height: 14px;
+  font-weight: 400;
+  color: #616070;
+  display: flex;
+  margin-left: 20px;
+  margin-top: 4px;
+  margin-bottom: 16px;
+  justify-content: space-between;
+}
+
+.card__rate {
+  width: 16px;
+  height: 16px;
+  background-image: url(/src/images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  margin-right: 4px;
+}
+
+.card__rate--star4:nth-child(-n + 5) {
+  background-image: url(/src/images/star-active.svg);
+}
+
+.card__rate--star:last-child {
+  margin-right: 0;
+}
+
+.card__rate--star5 {
+  display: flex;
+}
+
+.card__rate--review {
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: #060b35;
+  text-align: center;
+}
+
+.card__price {
+  justify-content: space-between;
+  display: flex;
+  width: 166px;
+}
+
+.card__price--item {
+  color: #616070;
+  font-weight: 400;
+  font-size: 12px;
+  line-height: 18px;
+}
+
+.card__price--value {
+  color: #060b35;
+  font-weight: 700;
+}
+
+.card__card__button {
```

Comment:
There seems to be a typo in the class name ".card__card__button". It should probably be ".card__button" to maintain consistency with the BEM naming convention.